server/form-pages/apply/assess-placement-risks-and-needs/behaviour-in-cas/previousStaysDetails.ts

```diff
@@ -10,7 +10,7 @@ const accommodationTypes = {
     text: 'Approved Premises (AP or CAS1)',
   },
   cas2: {
-    text: 'Bail Accommodation and Support Service (BASS or CAS2)',
+    text: 'CAS2 (formerly Bail Accommodation Support Services)',
```

Comment:
For my understanding: if we wanted to define the name of CAS2 singularly, do you know how we could do that?

In Rails I could use [the internationalisation API](https://guides.rubyonrails.org/i18n.html#the-public-i18n-api) as a way to give me a variable where we could say `I18n.t('service.identifier.cas2')`

Reply:
We could use an enum:

```ts
enum ServiceNames {
  CAS2 = "CAS2 (formerly Bail Accommodation Support Services)",
  CAS3 = "Transitional Accommodation (CAS3)"
}

const text = `Service name is ${ServiceNames.CAS2}`
```

Now that we've standardised Cas2 and Cas3, it'd be worth going through and refactoring. Shall I create a card?